src/App.js

```diff
@@ -1,23 +1,21 @@
-import logo from './logo.svg';
 import './App.css';
+import {Header} from "./components/Header";
+import { Route, Routes } from "react-router-dom";
+import {ArticleList} from "./components/ArticleList";
 
 function App() {
+
+
   return (
     <div className="App">
-      <header className="App-header">
-        <img src={logo} className="App-logo" alt="logo" />
-        <p>
-          Edit <code>src/App.js</code> and save to reload.
-        </p>
-        <a
-          className="App-link"
-          href="https://reactjs.org"
-          target="_blank"
-          rel="noopener noreferrer"
-        >
-          Learn React
-        </a>
-      </header>
+      <Header />
+      <Routes>
+        <Route
+          path="/"
+          element={<ArticleList />}
+        ></Route>
+        {/* <Route path="/:singleItem_slug" element={<SingleItem />}></Route> */}
```

Comment:
I know this is upcoming - don't include code that is not relevant to the current ticket